yt/utilities/command_line.py

```diff
@@ -1617,65 +1634,144 @@ def __call__(self, args):
+
+        if args.get("local", False):
+            config_file = local_config_file
+        elif args.get("global", False):
+            config_file = global_config_file
+        else:
+            if local_exists and global_exists:
+                s = (
+                    "Yt detected a local and a global configuration file, refusing "
+                    "to proceed.\n"
+                )
+                s += f"Local config file: {local_config_file}\n"
+                s += f"Global config file: {global_config_file}\n"
+                s += (
+                    "Specify which one you want to use using the `--local` or the "
+                    "`--global` flags."
+                )
+                sys.exit(s)
+            elif local_exists:
+                config_file = local_config_file
+            else:
+                config_file = global_config_file
+            sys.stderr.write(f"INFO: using configuration file: {config_file}.\n")
+
+        if not os.path.exists(config_file):
+            with open(config_file, "w") as f:
+                f.write("[yt]\n")
+
+        CONFIG.read(config_file)
+
+        self.config_file = config_file
+
+
+_global_local_args = [
+    (
+        "exclusive",
+        dict(
+            short="--local",
+            action="store_true",
+            help="Store the configuration in the global configuration file.",
+        ),
+        dict(
+            short="--global",
+            action="store_true",
+            help="Store the configuration in the global configuration file.",
+        ),
+    ),
+]
+
+
+class YTConfigGetCmd(YTCommand, YTConfigLocalConfigMixin):
     subparser = "config"
     name = "get"
     description = "get a config value"
-    args = (
-        dict(short="section", help="The section containing the option."),
-        dict(short="option", help="The option to retrieve."),
+    args = tuple(
+        [
+            dict(short="section", help="The section containing the option."),
+            dict(short="option", help="The option to retrieve."),
+        ]
+        + _global_local_args
     )
 
     def __call__(self, args):
         from yt.utilities.configure import get_config
 
+        self.load_config(args)
+
         print(get_config(args.section, args.option))
 
 
-class YTConfigSetCmd(YTCommand):
+class YTConfigSetCmd(YTCommand, YTConfigLocalConfigMixin):
     subparser = "config"
     name = "set"
     description = "set a config value"
-    args = (
-        dict(short="section", help="The section containing the option."),
-        dict(short="option", help="The option to set."),
-        dict(short="value", help="The value to set the option to."),
+    args = tuple(
+        [
+            dict(short="section", help="The section containing the option."),
+            dict(short="option", help="The option to set."),
+            dict(short="value", help="The value to set the option to."),
+        ]
+        + _global_local_args
```

Comment:
Maybe a matter of personal tastes
```suggestion
        dict(short="section", help="The section containing the option."),
        dict(short="option", help="The option to set."),
        dict(short="value", help="The value to set the option to."),
        *_global_local_args
```